package.json

```diff
@@ -63,12 +63,14 @@
     "commitizen": "3.0.5",
     "coveralls": "3.0.2",
     "cz-conventional-changelog": "2.1.0",
+    "dynalite": "^2.2.0",
     "dynamodb-local": "0.0.24",
     "eslint": "5.13.0",
     "husky": "1.3.1",
     "mocha": "5.2.0",
     "nyc": "13.1.0",
     "semantic-release": "15.13.3",
+    "shortid": "^2.2.14",
```

Comment:
i'd like to leave these two here, i believe there is a case for adding the id functionality to dynamoose, and honestly dynalite might be a nice change of pace for test speed